docs/source/Setup/Channels-to-Discord.md

```diff
@@ -0,0 +1,179 @@
+steps already, feel free to skip to the next.
+
+### Creating a Discord Bot Application
+
+> You will need an active Discord account and admin access to a Discord server
+> in order to connect Evennia to it. This assumes you already do.
+
+Make sure you're logged in on the Discord website, then visit
+https://discord.com/developers/applications. Click the "New Application"
+button in the upper right corner, then enter the name for your new app - the
+name of your Evennia game is a good option.
+
+You'll next be brought to the settings page for the new application. Click "Bot"
+on the sidebar menu, then "Build-a-Bot" to create your bot account.
+
+**Save the displayed token!** This will be the ONLY time that Discord will allow
+you to see that token - if you lose it, you will have to reset it. This token is
+how your bot confirms its identity, so it's very important.
+
+Next, add this token to your _secret_ settings.
+
+```python
+# file: mygame/server/conf/secret_settings.py
+
+DISCORD_BOT_TOKEN = '<token>'
+```
+
+Once that is saved, scroll down the Bot page a little more and find the toggle for
+"Message Content Intent". You'll need this to be toggled to ON, or you bot won't
+be able to read anyone's messages.
+
+Finally, you can add any additional settings to your new bot account: a display image,
+display nickname, bio, etc. You can come back and change these at any time, so
+don't worry about it too much now.
+
+### Adding your bot to your server
+
+While still in your new application, click "OAuth2" on the side menu, then "URL
+Generator". On this page, you'll generate an invite URL for your app, then visit
+that URL to add it to your server.
+
+In the top box, find the checkbox for `bot` and check it: this will make a second
+permissions box appear. In that box, you'll want to check off at least the
+following boxes:
+
+- Read Messages/View Channels (in "General Permissions")
+- Send Messages (in "Text Permissions")
+
+Lastly, scroll down to the bottom of the page and copy the resulting URL. It should
+look something like this:
+
+    https://discord.com/api/oauth2/authorize?client_id=55555555555555555&permissions=3072&scope=bot
+
+Visit that link, select the server for your Evennia connection, and confirm.
+
+After the bot is added to your server, you can fine-tune the permissions further
+through the usual Discord server administration.
+
+### Activating Discord in Evennia
+
+You'll need to do two additional things with your Evennia game before it can connect
+to Discord.
+
+First, install `pyopenssl` to your virtual environment, if you haven't already.
+
+    pip install pyopenssl
+
+Second, enable the Discord integration in your settings file.
+
+```python
+# file: server/conf/settings.py
+DISCORD_ENABLED = True
+```
+
+Start or reload your game to apply the changed settings, then log in as an account
+with at least `Developer` permissions and initialize the bot account on Evennia:
+
+    @discord2chan/name <your bot name>
```

Comment:
This breaks with the style of other bots; where the botname is hard-coded (such as grapewinehaus) or set directly in the definition, such as 

    irc2chan evennia_channel = <network_info> <botname>

Not _that_ big a deal, but it seems like it would be better to not invent yet another initialization style ...

Reply:
That's true! I originally had this in so you could customize how the relay is displayed, but since then I figured out a better solution to just take the bot name out of the evenni-side relay entirely. I don't mind taking it out for consistency; I'll make sure to remove the respective tests as well.